test/test_scaled_matmul_cuda.py

```diff
@@ -690,6 +690,10 @@ def _test_tautological_mm(self, device: str = "cuda",
             self.assertEqual(out_dtype, out_fp8.dtype)
         self.assertEqual(out_fp32, out_fp8.to(torch.float))
 
+    @unittest.skipIf(
+        _get_torch_cuda_version() >= (13, 0),
+        "Skip on CUDA 13.0+ due to known issues with FP8"
```

Comment:
Do we have an open issue I can look at for this? I took a real quick peek at open issues, nothing immediately jumped out at me